dvc/remote/base.py

```diff
@@ -257,8 +241,11 @@ def _download_dir(
                 )
             )
             with ThreadPoolExecutor(max_workers=self.remote.JOBS) as executor:
-                futures = executor.map(download_files, from_infos, to_infos)
-                return sum(futures)
+                executor.map(
```

Comment:
This is a tree method, so it also should raise an exception. Probably the first one that it gets, because it is similar to how you would implement it with `os.walk()`